src/lib/nav/DetailedMenu.svelte

```diff
@@ -17,7 +17,12 @@
   $: dve = $wlDetailedView;
 </script>
 
-<div class={`menu${$page.url?.pathname.startsWith("/search/") ? " on-search-page" : ""}`}>
+<div
+  class={`menu${$page.url?.pathname.startsWith("/search/") ? " on-search-page" : ""}`}
+  style="right: {$page.url?.pathname === '/' || $page.url?.pathname.startsWith('/search/')
```

Comment:
Was this to fix an issue? (I couldn't find if there was an issue with old code to review)